airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/StatePersistenceTest.java

```diff
@@ -55,6 +48,36 @@ class StatePersistenceTest extends BaseDatabaseConfigPersistenceTest {
   private static final String GLOBAL_STATE = "\"my global state\"";
   private static final String STATE = "state";
 
+  @BeforeEach
+  void beforeEach() throws DatabaseInitializationException, IOException, JsonValidationException, SQLException {
+    truncateAllTables();
+
+    setupTestData();
+    statePersistence = new StatePersistence(database);
+  }
+
+  private void setupTestData() throws JsonValidationException, IOException {
```

Comment:
this diff looks bigger than it is. this is just pulled up from lower down in the class.